plugin/adsk/scripts/USDMenuProc.mel

```diff
@@ -75,6 +87,11 @@ global proc USDMenuProc(string $parent, string $obj)
             menuItem -label "Edit As Maya Data" -image "edit_as_Maya.png" -command ("{waitCursor -state 1; mayaUsdMenu_editAsMaya \"" + $obj + "\"; waitCursor -state 0;}");
         }
         menuItem -label "Duplicate As Maya Data" -command ("{waitCursor -state 1; mayaUsdMenu_duplicate \"" + $obj + "\"; waitCursor -state 0;}");
+    
+        menuItem -divider true;
+
+        // Unassign (aka "unbind") the material from the selected object.
+        menuItem -label "Unassign Material" -command ("{waitCursor -state 1; mayaUsdMenu_unassignMaterial \"" + $obj + "\"; waitCursor -state 0;}");
```

Comment:
When you select this menu item what is shown as the command output in the Script Editor and is the action undoable?